core/pkg/eval/fractional_evaluation_test.go

```diff
@@ -55,13 +55,13 @@ func TestFractionalEvaluation(t *testing.T) {
 	}
 
 	tests := map[string]struct {
-		flags           Flags
-		flagKey         string
-		context         map[string]any
-		expectedValue   string
-		expectedVariant string
-		expectedReason  string
-		expectedError   error
+		flags             Flags
+		flagKey           string
+		context           map[string]any
+		expectedValue     string
+		expectedVariant   string
+		expectedReason    string
+		expectedErrorCode string
```

Comment:
maybe?
```suggestion
		expectedErrorMsg string
```
with code I expect an integer

Reply:
These are [openfeature error codes](https://openfeature.dev/specification/types#error-code) (which are particular strings, kinda like `ENOENT` or `EACCES` in unix).